src/wasm32/ffi.c

```diff
@@ -67,7 +67,6 @@ EM_JS_MACROS(
 void,
 ffi_call, (ffi_cif * cif, ffi_fp fn, void *rvalue, void **avalue),
 {
-  "use strict";
```

Comment:
This is useful for debugging purposes though. I guess we should add `-s STRICT_JS` to the linker flags.